### ISSUE TITLE
/lb clearlog does not work for chat messages

### ISSUE THREAD

**Issue by mb243** [LogBlock/LogBlock#662]
The SQL statement for log dumping generates an error under MySQL 5.7.

**Comment by md-5** [comment]
What's the error?

**Comment by mb243** [comment]
[20:48:44] [Craft Scheduler Thread - 9/ERROR]: [ClearLog] Exception while dumping log:
com.mysql.jdbc.exceptions.jdbc4.MySQLSyntaxErrorException: You have an error in your SQL syntax; check the manual that corresponds to y
our MySQL server version for the right syntax to use near 'save' in world.csv' FIELDS TERMINATED BY ',' OPTIONALLY ENCLOSED BY '"'  LIN
ES T' at line 1

**Comment by mb243** [comment]
Let me know if you need anything more and I'll do my best to find it for you :)

**Comment by md-5** [comment]
Can you please put that in a code block or pastebin, the formatting is mangled.

**Comment by mb243** [comment]
Edited. Let me know if that works for you.

**Comment by mb243** [comment]
This may not be an issue with MySQL 5.7 after all. The command I used was:

/lb clearlog chat search save

which I expected to search the chat log for the word "save" and clear all matches. This failed with the above. I then tried:

/lb clearlog chat search meow

which also generated the error. I then did 

/lb clearlog chat player <player>

which triggered a dump of "Block changes from player <player> in world.csv"

Either my understanding of the parameters of clearlog is incorrect, or it is not handling parameters correctly.

I
